### ISSUE TITLE
Packages.conifg filtering is undone every time a new package is installed

### ISSUE THREAD

**Issue by andrewjpoole** [OSSIndex/audit.net#12]
Every time a new package is installed via the nuget UI in visual studio, the xml comments seem to be removed, any chance the filter could look for an attribute rather than a comment?

**Comment by OSSIndex** [comment]
I agree that this seems like a good approach. We are looking into options and will fit the development into the schedule.

**Comment by OSSIndex-Admin** [comment]
So you know, this has not been forgotten. There is some code refactoring that will be performed shortly, and we will be adding in some advanced vulnerability filtering at the same time. The feature that I think will properly resolve your request is this one:

[Allow .devaudit configuration files to filter results](https://github.com/OSSIndex/audit.net/issues/13)

If I have misunderstood the issue or the feature does not otherwise fulfill your requirements please let me know.
